app/guid-file/styles.scss

```diff
@@ -56,29 +47,133 @@
     }
 }
 
-.FileDetail__revisions {
+.file-detail-revisions {
     max-height: 100vh;
     overflow-y: auto;
 }
 
-.FileDetail__right-section-heading {
+.file-detail-right-section-heading {
     margin-left: 20px;
 }
 
-.FileDetail__revision-list {
+.file-detail-revision-list {
     list-style-position: inside;
     padding-inline: 20px;
 }
 
-.FileDetail__no-revisions {
+.file-detail-no-revisions {
     padding: 20px;
     text-align: center;
 }
 
-.RightContainer {
+.right-container {
+    display: inline-flex;
+    flex-direction: column;
+    height: 95%;
     margin: 0 20px;
-    
+
+    dt {
+        font-size: 17px;
+        font-weight: 600;
+    }
+
+    dt,
+    dd {
+        background-color: #fff;
+        color: #000;
+        margin-bottom: 14px;
+    }
+
     h2 {
+        font-size: 31.5px;
+        font-weight: 300;
+        line-height: 1.1;
         margin-bottom: 20px;
     }
 }
+
+.metadata-heading {
+    margin-bottom: 0;
+}
+
+.node-type-heading {
+    border-top: 2px solid #ddd;
+    border-top: 2px solid rgb(221, 221, 221);
+    padding-top: 30px;
+}
+
+.metadata-title-edit-download {
+    align-items: baseline;
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+    white-space: nowrap;
+    width: 100%;
+
+    svg {
+        height: 1.5em;
+    }
+}
+
+.edit-download-buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 1vw;
+    grid-gap: 2vw;
+
+    a,
+    button {
+        padding: 0.7vw 1.1vw;
+    }
+
+    button:active,
+    button:focus,
+    button:hover,
+    a:active,
+    a:focus,
+    a:hover {
+        background: #efefef;
+    }
+}
+
+.metadata-form-and-buttons {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: space-between;
+}
+
+.edit-metadata-form {
+    display: inline-flex;
+    flex-direction: column;
+}
+
+.edit-metadata-save-cancel-buttons {
+    display: inline-flex;
+    flex-direction: row;
+    font-size: 14px;
+    grid-gap: 1vw;
+    justify-content: flex-end;
```

Comment:
The buttons end up too low on the screen if there's a tall document increasing the view height. Could we instead put the buttons just below the fields rather than at the bottom of the view?

Reply:
<img width="1619" alt="Screenshot 2023-01-12 at 9 40 38 AM" src="https://user-images.githubusercontent.com/6599111/212096016-a5c56830-f5e1-4d9a-9618-773b677b82ed.png">
<img width="1619" alt="Screenshot 2023-01-12 at 9 40 41 AM" src="https://user-images.githubusercontent.com/6599111/212096025-a15c9fdf-e26c-40a3-af91-d6f713e52893.png">

Reply:
Ok, these have been updated from the mock-ups and are now at the bottom of the form instead of the page. Thank you Brian.